creusot/src/translation/constant.rs

```diff
@@ -162,6 +163,11 @@ fn try_to_bits<'tcx, C: ToBits<'tcx>>(
             }
         }
         _ if ty.is_unit() => Exp::Tuple(Vec::new()),
+        FnDef(def_id, subst) => {
+            let method = resolve_assoc_item_opt(ctx.tcx, env, *def_id, subst).unwrap_or((*def_id, subst));
```

Comment:
unneeded

Reply:
Ok then why are there calls to `resolve_assoc_item_opt` here and there in the translation?

Reply:
Yea, I guess we still need to do this manually here. 

This is one of the things I would like to clean up by introducing an actual IR.

Reply:
So should I leave this or not?

Reply:
Leave it.